src/utils.js

```diff
@@ -172,7 +172,7 @@ function getNullableText(...args) {
         if (arg)
             return arg;
 
-        if (arg === false || arg === null)
+        if (arg === false || arg === null || arg === '')
```

Comment:
This should have been added to the `if (arg)` condition. I would consider an empty string to be a valid value that people  might want to set.

Reply:
updated